### ISSUE TITLE
Eva: API In Progress

### ISSUE THREAD

**Issue by evagravin** [evagravin/p4-kangaroos#35]

**Comment by avabrooks** [comment]
Great ticket, API is necessary and DN group is working now so should be easier to connect to db. I was working on the most recent smack and was having trouble getting certain functions to work on the home page as well similar to what you were experiencing with the quote of the day.

**Comment by risaiwazaki** [comment]
Yes the API is really important so nice job on getting that figured out!! I took a look at the quote of the day api and I did some research on the error that it showed... I'll send you a couple links and maybe we could figure it out together :)

**Comment by lindalonglong** [comment]
Nice ticket, I believe API with database is necessary for the smack DM. I was working on the message interface and also figuring out how to connect it with DN users.
